pkg/image/docker/daemon_provider.go

```diff
@@ -182,9 +197,6 @@ func (p *DaemonImageProvider) Provide() (*image.Image, error) {
 		}
 	}()
 
-	// cancel indeterminate progress
-	estimateSaveProgress.SetCompleted()
```

Comment:
this should probably stick around. Why? The image save progress is only a guess (a timer counting up to a particular time where the overall progress would be considered at 50%). It's logical to adjust the first image save timer to complete when the image save operation returns.